src/clj/quil/applet.clj

```diff
@@ -66,7 +66,6 @@
   (let [m              (meta applet)
         keep-on-top?   (:keep-on-top m)
         surface        (.getSurface applet)
-        frame          (.frame applet)
```

Comment:
frame wasn't being used and has also been [removed](https://github.com/processing/processing4#alpha-2)